AirCasting/Dashboard/DashboardView.swift

```diff
@@ -14,15 +14,16 @@ import Resolver
 struct DashboardView: View {
     @StateObject var coreDataHook: CoreDataHook
     @FetchRequest<SensorThreshold>(sortDescriptors: [.init(key: "sensorName", ascending: true)]) var thresholds
-    @EnvironmentObject var selectedSection: SelectSection
+    // TODO: - We can remove it in a future
```

Comment:
Could you be more specific in this comment? When exactly can this be removed?

Reply:
Pawel told me that in the future we can rethink the way how it's working and change it.

Reply:
In what future? If we will not do it in the next few days, it would be good to change this comment, so that it states specifically what can be removed and when. What needs to happen for this to be removed? We already have some cryptic warning messages that we don't know what they mean.